tests/plots/test_util.py

```diff
@@ -154,3 +156,26 @@ def test_gempak_color_scalar():
     mplc = convert_gempak_color(6)
     truth = 'cyan'
     assert mplc == truth
+
+
+@pytest.mark.mpl_image_compare(remove_text=True)
+def test_plot_extrema():
+    """Test plotting of local max/min values."""
+    data = xr.open_dataset(get_test_data('GFS_test.nc', as_file_obj=False))
+
+    mslp = data.Pressure_reduced_to_MSL_msl.squeeze()
+    relmax2d = find_local_extrema(mslp, 10, 'max').metpy.convert_units('hPa')
+    relmin2d = find_local_extrema(mslp, 15, 'min').metpy.convert_units('hPa')
+
+    fig = plt.figure(figsize=(8., 8.))
+    ax = fig.add_subplot(1, 1, 1)
+
+    # Plot MSLP
+    clevmslp = np.arange(800., 1120., 4)
+    ax.contour(mslp.lon, mslp.lat, mslp.metpy.convert_units('hPa'),
+               clevmslp, colors='k', linewidths=1.25, linestyles='solid')
```

Comment:
I don't like having a completely unrelated line in the test code, but this really helps make the test useful for humans so 👍 .

Reply:
yeah, I thought about that when making the test and I decided to keep them because of the human sanity check in the event something went wrong there was an extra way to indicate what *should* be the correct answer.